test/task.test.js

```diff
@@ -0,0 +1,21 @@
+const Task = require('../src/taskmodel');
+const assert = require('assert').strict
+
+describe('integration test', function () 
+{
+    describe('task tests', function() 
+    {
+        describe('id tests', function() 
```

Comment:
Meinem Verständnis nach müssen nur für Item id tests geschrieben werden. Weil id und Funktionen für id nicht in Task definiert werden. Task erbt von id Funktionalität von Item. Wenn alles mit id in Item rund läuft, dann funktionert es auch in Task.